02-oop-inheritance/javaScript/downloableMovie.js

```diff
@@ -0,0 +1,59 @@
+function MovieObserver() {
+
+  this.playing = function (title) {
+    console.log('A movie with title ' + title + ' has been played');
+  } 
+  this.stopped = function(title) {
+    console.log('A movie with title ' + title + ' stopped');
+  } 
+}
+
+function Movie() {
```

Comment:
also you should leave a white space between the function name and the parenthesis